install.sh

```diff
@@ -63,9 +63,25 @@ install_extension() {
         mkdir -p /tmp/extensions/resources
     fi
     cp -Rf resources/* /tmp/extensions/resources/
+
+    if [ -n "$ext_vars" ] && [ -n "$ext_name" ]; then
+        create_extension_js_file_with_vars
+    fi
+
     echo "UI extension installed successfully"
+
 }
 
+create_extension_js_file_with_vars() {
+  echo "Generating extension vars js file..."
+  ext_js_file_path="/tmp/extensions/resources/extension-$ext_name.js"
+  ext_js_file_name="vars-$(date +"%Y%m%d%H%M%S")"
```

Comment:
Shouldn't we replace the file every time instead of creating multiple ones with different timestamps in the name?

Reply:
make sense, no need to create  dynamic  filename. Restart can create multiple file,  however it shouldn't cause any conflict to variable but it is unnecessary 

add the constant name
`  ext_js_file_name="$EXTENSION_NAME-vars"
`